examples/middleware/tsconfig.json

```diff
@@ -21,5 +21,5 @@
     ]
   },
   "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
-  "exclude": ["node_modules", "open-next.config.ts", "worker.ts"]
+  "exclude": ["node_modules"]
```

Comment:
question: why is this file and its permission updated?

Reply:
I made this change by mistake, I was sure I reverted it but I forgot to push the change up, done, thanks for noticing it 👍